### PR TITLE
Training popups: rebased for 16.0

### PR DESCRIPTION
**Pull request by taracampbell**
Feature request from CCNA. The code is a mess right now - still a work in progress before it can be merged into LORIS. I just put everything up here for now for CCNA team reference.

Feature: Quiz has extra information displayed after you respond. It shows you which questions you got wrong and right, highlights the correct answers, and explains why the questions you got wrong are wrong.

screen shot 2016-05-12 at 3 39 45 pm

### CODE REVIEW

**Comment by alisterdev** [comment]
@taracampbell When answering question wrong multiple times, the correct answer / explanation is appended multiple times instead of just once. See screenshot below

image

**Comment by alisterdev** [comment]
@taracampbell clicking agree on a video tab, doesn't stop the video before going to the next tab ( so there is video sound still playing on all following tabs)

**Comment by alisterdev** [comment]
@taracampbell In completed certifications, quiz tab still shows but has no option to either:
a) Redo/resubmit the quiz or
b) See your past results and/or correct answers

**Comment by alisterdev** [comment]
As a side note, some tabs (e.g those with videos) can take a few seconds to load. It is not clear to the user what is happening at that moment. So, as a suggestion, we can incorporate a loading spinner that is shown while tab content is loading (i.e http://projects.lukehaas.me/css-loaders/)

**Comment by taracampbell** [comment]
@alisterdev For some reason the quiz button was no longer being disabled after submission. I've fixed the javascript so that users can't click on it multiple times, fixing the answer explanation appending for now.

Other issues you mention will probably be fixed in the next version of the training module... they're nice, but not necessary for our impending June 1st release,

**Comment by alisterdev** [comment]
@taracampbell disabling submit button is not really a viable solution, because if a user does not respond correctly from the first try, he could not try to answer again and thus would never be able to complete the training module

**Comment by taracampbell** [comment]
@alisterdev Hey Alex, they get certified no matter what they answer (feature requested by CCNA team). The point is just for them to learn why the answer they chose as wrong is wrong, and to show which the correct answer is. Plus it shows you the right answer anyway, so not much point getting them to take the quiz once they fail it.

**Comment by alisterdev** [comment]
@taracampbell I see, but in this case there should a clear UI indicating that they don't have to answer correctly to be certified. The way it is right now, it is not clear what to do if you answered wrong, since there is no way to go back/forward. Perhaps a back button should be added to get back to training module.

File: modules/training/ajax/markQuiz.php [hunk 1]
```diff
@@ -29,41 +29,33 @@
...
+           'date_cert' => array($instrumentID => $dateArray),
+           'examiner'  => $examinerID,
+          );
 
-    process($values);
-
-    print "correct";
-    exit();
-}
+//process($values);
```
Comment (alisterdev) [inline modules/training/ajax/markQuiz.php]: @taracampbell This shouldn't be commented out I believe. When process is not called the completed certifications are not moved from "Certifications to complete" to "Completed Certifications"

**Comment by tedstrauss** [comment]
The quiz seems to be broken, only a single response option is showing. 
screenshot from 2016-06-08 18 13 18

**Pull request closed without merging**